packages/shared/schemas/src/lib/domain/character_book.ts

```
@@ -0,0 +1,66 @@
+
+// ── CharacterBookEntrySchema ──────────────────────────────────────────────
+
+export const CharacterBookEntrySchema = Type.Object({
+  keys: Type.Array(Type.String(), { description: 'Keywords that trigger this entry' }),
+  content: Type.String({ description: 'The lore content to inject' }),
+  extensions: Type.Record(Type.String(), Type.Unknown(), {
+    description: 'Vendor-specific extensions (preserved verbatim)',
+    default: {},
+  }),
+  enabled: Type.Boolean({ description: 'Whether this entry is active', default: true }),
+  insertion_order: Type.Integer({ description: 'Order/priority within the book' }),
```

Biome (2.5.7)

[error] This object property name should be in camelCase.

(lint/style/useNamingConvention)